packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -40,26 +95,34 @@ export class SankeyChartBase extends React.Component<
       theme: theme!,
       width: this.state.containerWidth,
       height: this.state.containerHeight,
-      pathColor: pathColor,
+      pathColor,
       className,
     });
-    const margin = { top: 10, right: 0, bottom: 10, left: 0 };
-    const width = this.state.containerWidth - margin.left - margin.right;
-    const height =
-      this.state.containerHeight - margin.top - margin.bottom > 0
-        ? this.state.containerHeight - margin.top - margin.bottom
-        : 0;
+    const margin = { top: 36, right: 48, bottom: 32, left: 48 };
+    // We are using the margin.left and margin.top in sankey extent while constructing the layout
+    const width = this.state.containerWidth - margin.right;
+    const height = this.state.containerHeight - margin.bottom > 0 ? this.state.containerHeight - margin.bottom : 0;
 
     const sankey = d3Sankey
       .sankey()
-      .nodeWidth(5)
-      .nodePadding(6)
+      .nodeWidth(124)
```

Comment:
This can be customized, but this is mentioned as constant in design. If we change it then we might need to change few conditions for truncation of text.